internal/sail/client/roomer.go

```diff
@@ -31,7 +31,7 @@ func (r httpRoomer) NewRoom(ctx context.Context, version model.WebVersion) (room
 
 	resp, err := http.Post(u.String(), "text/plain", bytes.NewReader(reqJson))
 	if err != nil {
-		return "", "", errors.Wrapf(err, "GET %s", u.String())
+		return "", "", err
```

Comment:
removed this wrapper because
1) This isn't even a GET, so the message was wrong
2) http.Post errors already include "post [url]", so this led to weird duplication